spec/classes/backend_spec.rb

```diff
@@ -108,11 +108,12 @@
 
         it {
           should contain_file('sensu_backend_config').with({
-            'ensure'  => 'file',
-            'path'    => '/etc/sensu/backend.yml',
-            'content' => backend_content,
-            'require' => 'Package[sensu-go-backend]',
-            'notify'  => 'Service[sensu-backend]',
+            'ensure'    => 'file',
```

Comment:
there should be a test that demonstrates when `show_diff` is set to `false`.

Reply:
Test added